src/world/sprite.h

```diff
@@ -201,7 +201,7 @@ typedef struct rct_money_effect {
 	uint8 linked_list_type_offset;	// 0x08 Valid values are SPRITE_LINKEDLIST_OFFSET_...
 	uint8 var_09;
 	uint16 sprite_index;			// 0x0A
-	uint16 flags;			// 0x0C
+	uint16 flags;					// 0x0C (Flags on money_effects do not seem to be used, so it is used for string_id)
```

Comment:
The potential problem with using flags on a money effect is it having to check whether or not it is grey each time it is drawn. i feel storing the info it only needs once is slightly better than having to store info and then check against that info each time. i know it's a miniscule speed difference but considering it can have many drawn on the screen at once, it could hinder performance. plus, the money effect is so minor of a feature i doubt it would ever need flags.

your ideas?